src/Components/DropdownField.res

```diff
@@ -24,13 +43,31 @@ let make = (
     let target = ev->ReactEvent.Form.target
     let value = target["value"]
     setValue(value)
+    if isDisplayValueVisible {
+      let findDisplayValue =
+        options
+        ->Array.find((ele: optionType) => ele.value === value)
```

Comment:
You can remove the type from this, the code will work even with `->Array.find(ele => ele.value === value)`

Reply:
Updated